### DISCUSSION TITLE
Y2 and Y2[-1] Showing same value

### DISCUSSION THREAD

**Discussion by PKeyes505** [artisan-roaster-scope/artisan#692] [General]
I am trying to follow along with the blog post below.  Why is P1 (Y2) and P2 (Y2[-1]) the same value,  yet in P3 (Y2-Y2[-1]) I get the expected difference? 
 
https://artisan-roasterscope.blogspot.com/2015/10/signals-symbolic-assignments-and-plotter.html

![image](https://user-images.githubusercontent.com/61755459/133841434-e518eb0f-47ff-4e65-9de8-10def317e709.png)

Thank you

**Comment by MAKOMO** [comment]
> Why is P1 (Y2) and P2 (Y2[-1]) the same value, yet in P3 (Y2-Y2[-1]) I get the expected difference?

There was a typo in the code computing the plotter data table. Fixed in source. Thanks for reporting!

Your P8 formula contains a typo (an extra closing bracket). Instead of

`(Y2-Y2[-1])/(t-t[-1]))*60*k+o`

it should be

`(Y2-Y2[-1])/(t-t[-1])*60*k+o`

New builds are produced right now and should be available in about half an hour here:
https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous

<img width="1063" alt="Screenshot 2021-09-18 at 11 46 46" src="https://user-images.githubusercontent.com/13909011/133884873-a2998d8d-b1ab-4c9f-9d42-6d2788bff67c.png">

**Reply by PKeyes505** [reply]
Thank you for reading my post, the fix and the reply.  Very cool.
